scripts/gha/ui_testing/uitest_android/app/src/androidTest/java/com/google/firebase/uitest/UITest.java

```diff
@@ -127,7 +132,28 @@ public void testGMA() throws UiObjectNotFoundException, InterruptedException {
   private void launchApp(String packageName) {
     Context context = getApplicationContext();
     Intent intent = context.getPackageManager().getLaunchIntentForPackage(packageName);
+    intent.setAction("com.google.intent.action.UI_TEST");
+    intent.addCategory(Intent.CATEGORY_DEFAULT);
+    intent.setType("application/javascript");
```

Comment:
What does this intent have to do with javascript?

Reply:
To send data with intent here
https://github.com/firebase/firebase-cpp-sdk/blob/feature/app-framework-uitest/scripts/gha/ui_testing/uitest_android/app/src/androidTest/java/com/google/firebase/uitest/UITest.java#L152
The type `javascript ` chosen here is to make it consistent with our existing game-loop test:
https://firebase.google.com/docs/test-lab/android/game-loop#begin-anchor